### DISCUSSION TITLE
New - Promo Card variation

### DISCUSSION THREAD

**Discussion by IdaHW** [Legal-and-General/canopy#728] [Canopy requests]
Type of request
New

Component, pattern or foundation?
New settings to the existing Promo Card component

Nature of request
As part of the Dotcom frontend rebuild we need variations to the existing Promo Card to be added to Canopy to allow for the designs below.

First setting would allow for the image either on the right or left:
 
![Screenshot 2022-03-15 at 15 04 03](https://user-images.githubusercontent.com/97164118/158415600-eb52c3b4-9c66-4402-b6a8-fd04a2071b6b.png)

Second setting would modify the content layout/positioning and an option to add a border, to allow for the following design/layout:

![Screenshot 2022-03-15 at 15 05 14](https://user-images.githubusercontent.com/97164118/158423793-e6034b71-341a-43e7-a0e7-8f3bde171ffe.png)

**Comment by elenagarrone** [comment]
@IdaHW apologies for the late reply - we didn't get a notification for the discussion. We are working on fixing that so that we can be more responsive but feel free to nudge me or Alex on the Canopy channel or fep-support.

Existing promo card:
![Screenshot 2022-03-17 at 13 50 29](https://user-images.githubusercontent.com/8397116/158823352-c2684055-d576-4614-a8e7-88584c875331.png)

I guess my question is for the designers (@alexcanning) - do we need a new variant of the promo card? can we use the existing one or maybe update it so that it looks like the new suggested one instead of having both?
The only difference is how the content is positioned in relation to the image.

I guess that's the main question and based on the answer we can decide what to do with the horizontal. Just FYI on small devices the existing `promo-card` looks like this:
![Screenshot 2022-03-17 at 13 53 34](https://user-images.githubusercontent.com/8397116/158823998-d2f633a8-fafe-4096-8b15-c1420aa12338.png)

**Reply by alexcanning** [reply]
@james-hayden-ellis it would make sense to modify the existing promo card component as Elena notes, but bearing in mind that whatever you decide works stylistically for Dotcom, would also need to work for self-service.

**Reply by james-hayden-ellis** [reply]
We need to bear in mind that these designs are based on existing Epi-Components so have prerequisite features and functions. The top component in the design is a [Dual Box](https://landg.atlassian.net/wiki/spaces/CETCE/pages/996317344/Boxes#DUAL-BOX-BLOCK) and is itself comprised of other Epi-Components and is therefore distinctly different from the component below. The [Box Blocks](https://landg.atlassian.net/wiki/spaces/CETCE/pages/996317344/Boxes#BOX-BLOCK) beneath are closer in keeping with the Promo Cards but we may want to differentiate between the different types of content to make it easier on the CMS / Content Editor maybe?

**Comment by alexcanning** [comment]
Closing as this has been superseded by a new promo card pattern and Dotcom components.
